apps/chat-e2e/src/assertions/api/apiAssertion.ts

```diff
@@ -72,7 +72,7 @@ export class ApiAssertion {
 
   public async assertRequestPrompt(request: ChatBody, expectedPrompt: string) {
     expect
-      .soft(request.prompt, ExpectedMessages.chatRequestPromptIsValid)
+      .soft(request.prompt ?? '', ExpectedMessages.chatRequestPromptIsValid)
```

Comment:
What if the prompt is intentionally missing? Can we have a case like that? It might mask some cases

Reply:
Can we consider adding a separate assertion to check for the existence of the prompt separately, before this assertion?

Reply:
if the prompt is not set or empty, it is not sent in the request. I'll add additional check to cover such cases